konrad/core.py

```diff
@@ -223,6 +233,15 @@ def run(self):
                 timestep=self.timestep,
             )
 
+            if self.is_co2_adjusting:
+                # adjust CO2 concentrations to find a equilibrium state using
+                # equation 8 of Romps 2020
+                n0 = self.surface.heat_sink
+                A = 5.35
+                tau = 1
```

Comment:
Should it be possible for the user to modify these?

Reply:
These are the values taken from Romps 2020; he argues why these are reasonable values to use, but in theory it doesn't matter what they are. They adjust the timescale of CO2 adjustment and if we're doing CO2 adjustment, we are doing it because we only care about the equilibrium state.

For my experiments, when I started relatively close to equilibrium (eg I started from a 1xCO2 equilibrium state and increased the surface temperature a bit), these values were fine. However, when starting further from equilibrium, values which make the CO2 adjustment occur more slowly might be needed. So in that case, maybe it would be good to allow the user to modify them (or one of them) and just use those values as defaults.

Reply:
I am totally fine with the value of `A` as it is justified in the paper (and it is sufficient to tune one variable). But I though he is arguing for a `tau` of "one week" or does he modify that later?
Anyhow, I just thought that a small timestep might cause problems in the stratosphere as a rapid cooling sometimes leads to this oscillating temperature profile patterns that tend to explode ;)

Reply:
Ahh, I think I got our time units wrong, I meant it to be one week!

Reply:
I guess it is fine to hard-code it at to one week for now. Maybe leave a `TODO` comment that it might be valuable to make this variable at some point